xsls/body.xsls

```diff
@@ -18,20 +18,27 @@ X:template body (lang) {
     <body>
 
     <div id="banner">
+        <div id="banner-content"></div>
     </div>
 
     <div id="main">
-    <div id="menu">
+    <div id="menu" >
```

Comment:
Is it a space coming from line 28/32 ?

Reply:
Fixed with https://github.com/nginx/nginx.org/pull/20/commits/c6ae5f0ff6590e087697b8ce255b90620933017f (will resubmit the PR eventually).